test/VM_Test/playlist.xml

```diff
@@ -78,44 +78,6 @@
 
 	<test>
 		<featureIds>
-			<featureId>113758</featureId>
-		</featureIds>
-		<testCaseName>recreateClassTest</testCaseName>
-		<variations>
-			<variation>NoOptions</variation>
-		</variations>
-		<command>
-	$(JAVA_COMMAND) $(JVM_OPTIONS) \
-	-javaagent:$(Q)$(TEST_RESROOT)$(D)VM_Test.jar$(Q) \
-	-cp $(Q)$(LIB_DIR)$(D)junit4.jar$(P)$(LIB_DIR)$(D)asm-all.jar$(P)$(TEST_RESROOT)$(D)VM_Test.jar$(P)$(LIB_DIR)$(D)javassist.jar$(Q) \
-	junit.textui.TestRunner com.ibm.j9.recreateclass.tests.RecreateClassTestSuite; \
-	$(TEST_STATUS)</command>
-		<tags>
-			<tag>sanity</tag>
-		</tags>
-		<subsets>
-			<subset>SE80</subset>
-		</subsets>
-	</test>
-
-	<test>
-		<featureIds>
-			<featureId>133610</featureId>
-		</featureIds>
-		<testCaseName>CFdumptest</testCaseName>
-		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
-	-cp $(Q)$(TEST_RESROOT)$(D)VM_Test.jar$(P)$(LIB_DIR)$(D)junit4.jar$(Q) \
-	junit.textui.TestRunner com.ibm.j9.cfdump.tests.CfdumpTestSuite; \
-	$(TEST_STATUS)</command>
-		<tags>
-			<tag>sanity</tag>
-		</tags>
-		<subsets>
-			<subset>SE80</subset>
-		</subsets>
-	</test>
-	<test>
-		<featureIds>
```

Comment:
Please create another file and store the removed tests above into it (i.e., excluded_playlist.xml). We are working on generate .so for these tests in OpenJ9. Eventually, these tests will need to re-included and having excluded_playlist.xml is easier for us to re-included these tests back. Thanks.

Reply:
Good idea. I've added a test/VM_Test/excluded_playlist.xml file.